bitcoin/src/blockdata/witness.rs

```diff
@@ -20,14 +20,14 @@ use crate::VarInt;
 /// The Witness is the data used to unlock bitcoins since the [segwit upgrade](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki)
 ///
 /// Can be logically seen as an array of byte-arrays `Vec<Vec<u8>>` and indeed you can convert from
-/// it [`Witness::from_vec`] and convert into it [`Witness::to_vec`].
+/// it [`Witness::from_slice`] and convert into it [`Witness::to_vec`].
```

Comment:
The wording was already weird, should be "from it using `Witness::from_slice` and convert into it using `Witness::to_vec`"